### PR TITLE
Feature/prettier

### PR DESCRIPTION
**Pull request by ColinKrist**
Are you tired of having to do `CTRL + SHIFT + P` then run `ESLINT: Auto Fix All Problems` like 10 times when doing a massive code change? Well do I have the solution for you my friends.

Now whenever you are in this repository, (after you re-run yarn of course) and install the `Prettier` plugin in VSCode you will get auto formatting of your code that runs off the new .prettierrc and eslint config we already have. Neato!

I tried getting the styling as close to what we had as possible, some of the stuff had to change though cause it wasn't supported by prettier apparently. Nothing major.

Let me know if the line 100 length is ok, from what I saw anything > 100 in .jsx files looked stupid and was hard to read cause all the props just went on and on after formatting.

### CODE REVIEW

File: zoo_frontend/src/pages/food/units/units.jsx [hunk 1]
```diff
@@ -16,7 +16,9 @@ export default class units extends Component {
   static async getInitialProps({ authToken }) {
     const serverUnitsAPI = new UnitsAPI(authToken);
     try {
-      const res = await serverUnitsAPI.getUnits().catch((err) => ({ data: [{ err: true, msg: err }] }));
+      const res = await serverUnitsAPI
+        .getUnits()
+        .catch((err) => ({ data: [{ err: true, msg: err }] }));
```
Comment (MitchellSlavik) [inline zoo_frontend/src/pages/food/units/units.jsx:21]: Personally I find this kind of stuff harder to read than the original

File: zoo_frontend/src/pages/profile/profile.jsx [hunk 2]
```diff
@@ -79,43 +79,30 @@ class Profile extends Component {
...
       }
     }
-  }
+  };
 
   onOldPasswordTextChange = (event) => this.setState({ oldPasswordText: event.target.value });
   onNewPasswordTextChange = (event) => this.setState({ newPasswordText: event.target.value });
-  onConfirmNewPasswordTextChange = (event) => this.setState({ confirmNewPasswordText: event.target.value });
+  onConfirmNewPasswordTextChange = (event) =>
+    this.setState({ confirmNewPasswordText: event.target.value });
```
Comment (MitchellSlavik) [inline zoo_frontend/src/pages/profile/profile.jsx:87]: (eye roll)

File: zoo_frontend/src/util/AuthProvider.jsx [hunk 3]
```diff
@@ -35,11 +32,7 @@ class AuthProvider extends Component {
   }
 
   render() {
-    return (
-      <AuthContext.Provider value={this.state}>
-        {this.props.children}
-      </AuthContext.Provider>
-    );
+    return <AuthContext.Provider value={this.state}>{this.props.children}</AuthContext.Provider>;
```
Comment (MitchellSlavik) [inline zoo_frontend/src/util/AuthProvider.jsx:35]: This is also more tiresome to read than the original

File: zoo_frontend/src/util/withAuth.jsx [hunk 4]
```diff
@@ -114,13 +114,7 @@ export default (allowedRoles = ['authenticated']) => (WrappedComponent) => {
...
     render() {
       if (!this.state.loading) {
-        return (
-          <WrappedComponent
-            {...this.props}
-            account={this.state.account}
-            api={this.api}
-          />
-        );
+        return <WrappedComponent {...this.props} account={this.state.account} api={this.api} />;
```
Comment (MitchellSlavik) [inline zoo_frontend/src/util/withAuth.jsx:117]: Here too, I also dont like how it removes the parenthesis around the jsx.

**Review by buffginger — APPROVED** [review]
Approving, assuming all the changes are formatting. 100 lines seems to be standard, I'm ok with that.

**Comment by ColinKrist** [comment]
I'll probably close this since I don't think I can solve Mitchell's
requests with the framework.

**Pull request closed without merging**